composer.json

```diff
@@ -187,6 +187,9 @@
         "su-sws/stanford_profile_helper": "^9.10",
         "su-sws/stanford_samlauth": "^1.0"
     },
+    "conflict": {
+        "drupal/migrate_tools": ">6.0.5"
+    },
```

Comment:
Lets remove this. if there's still an issue, i'll work on a solution.

Reply:
that's removed.